x-pack/filebeat/module/panw/panos/test/pan_inc_threat.log-expected.json

```diff
@@ -1516,7 +1616,10 @@
             "pan-os",
             "forwarded"
         ],
-        "url.original": "cls-softwares.com/softwarefortubeview.40013.exe"
+        "url.domain": "cls-softwares.com",
+        "url.extension": "40013.exe",
```

Comment:
I think for extensions we just want the last one.  file.extension has that recommendation, but url.extension doesn't explicitly say that in ECS spec.  What do you think?

Reply:
To me makes sense to be consistent with file.extension and just keep the last part